CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,3 @@
 # Code of Conduct
 
-All code and communication under WPT is covered by the [W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/).
+All code and communication under WPT is covered by the [W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/). See also the [Positive Work Environment Home Page](https://www.w3.org/Consortium/pwe/#Procedures).
```

Comment:
I'd suggest something more like, "For details on who to contact, see [here]."

Reply:
done